server/src/contact/contact.controller.ts

```diff
@@ -1,6 +1,9 @@
-import { Controller, Session, Get, UseInterceptors, HttpException, HttpStatus } from '@nestjs/common';
+import { Controller, Session, Get, UseInterceptors, HttpException, HttpStatus, Post, Body } from '@nestjs/common';
```

Comment:
Looks like we can delete the unused `Post`, `Body`, `import { pick } from 'underscore';` and also `import { CONTACT_ATTRS } from './contacts.attrs';`